spec/factories/chargeback_rate_detail_currency.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :chargeback_rate_detail_currency do
+    code  "EUR"
```

Comment:
Is there a reason to split this factory from the one below?  I don't see this one used and the factory below :point_down:  is for the Euro so I don't understand this one.  Again, we can clean this up in a follow up PR afterwards.